lib/msf/base/sessions/meterpreter_options.rb

```diff
@@ -71,6 +71,14 @@ def initialize(info = {})
               'AutoUnhookProcess',
               [true, "Automatically load the unhook extension and unhook the process", false]
             ),
+            OptBool.new(
+              'MeterpreterDebugBuild',
+              [true, 'Use a debug version of Meterpreter', false]
```

Comment:
Why would this be a `required` option? If not set then we would just consider it false.